movie_REST_API/movie_REST_API/AssemblyBuilder/AssemblyBuilder.swift

```diff
@@ -0,0 +1,31 @@
+// AssemblyBuilder.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Протокол для AssemblyBuilder
+protocol AssemblyBuilderProtocol {
+    func createMovieModule() -> UIViewController
+    func createInfoMovieModule(movie: Movie) -> UIViewController
+}
```

Comment:
```suggestion
/// Протокол для AssemblyBuilder
protocol AssemblyBuilderProtocol {
    func createMovieModule() -> UIViewController
    func createInfoMovieModule(movie: Movie) -> UIViewController
}
```
протокол в отдельный файл пожалуйста